components/brave_extension/extension/brave_extension/background/events/cosmeticFilterEvents.ts

```diff
@@ -28,13 +28,6 @@ chrome.contextMenus.create({
   id: 'brave',
   contexts: ['all']
 })
-// block ad child menu
-chrome.contextMenus.create({
-  title: getLocale('addBlockElement'),
-  id: 'addBlockElement',
-  parentId: 'brave',
-  contexts: ['all']
-})
 chrome.contextMenus.create({
```

Comment:
My only other suggestion would be to keep the original ordering of the menu items, otherwise LGTM